Sources/Controllers/2.0/Settings/OACheckForProfileDuplicatesViewController.mm

```diff
@@ -0,0 +1,254 @@
+//
+//  OACheckForProfileDuplicatesViewController.mm
+//  OsmAnd Maps
+//
+//  Created by Anna Bibyk on 19.10.2020.
+//  Copyright © 2020 OsmAnd. All rights reserved.
+//
+
+#import "OACheckForProfileDuplicatesViewController.h"
+#import "OAImportDuplicatesViewController.h"
+#import "OAImportCompleteViewController.h"
+#import "OAActivityViewWithTitleCell.h"
+#import "OASettingsHelper.h"
+#import "OASettingsImporter.h"
+#import "OAAppSettings.h"
+#import "OsmAndApp.h"
+#import "OAQuickAction.h"
+#import "OAPOIUIFilter.h"
+#import "OAMapSource.h"
+#import "OAResourcesUIHelper.h"
+#import "OAAvoidRoadInfo.h"
+#import "Localization.h"
+#import "OAColors.h"
+
+#define kSidePadding 16
+#define kTopPadding 6
+#define kBottomPadding 32
+#define kCellTypeWithActivity @"OAActivityViewWithTitleCell"
+
+@interface OACheckForProfileDuplicatesViewController() <UITableViewDelegate, UITableViewDataSource, OASettingsImportExportDelegate>
+
+@end
+
+@implementation OACheckForProfileDuplicatesViewController
```

Comment:
Doesn't seem to be ported from ImportDuplicatesFragment.java (please sync the code, rename methods to be identical with android where possible)